src/main.ts

```diff
@@ -370,6 +374,8 @@ export default class LinterPlugin extends Plugin {
     } catch (error) {
       this.handleLintError(file, error, 'Lint File Error in File \'${file.path}\'', false);
     }
+
+    this.rulesRunner.runCustomRegexReplacement(this.settings.customRegexs, editor);
```

Comment:
I would actually have the custom regex rules run at the end of the regular rules in the rulesRunner when a file gets linted.